performance/build.gradle

```diff
@@ -146,14 +148,41 @@ private def getCommonProperties() {
             'kotlinVersion': "${kotlinVersion}".toString()]
 }
 
+private def uploadBenchmarkResultToBintray(String fileName) {
+    def teamcityConfig = System.getenv("TEAMCITY_BUILD_PROPERTIES_FILE")
+    if (teamcityConfig) {
+        def buildProperties = new Properties()
+        buildProperties.load(new FileInputStream(teamcityConfig))
+        def user = buildProperties.getProperty("bintray.user")
+        def password = buildProperties.getProperty("bintray.apikey")
+        def buildNumber = buildProperties.getProperty("build.number")
+        def target = System.getProperty("os.name").replaceAll("\\s", "")
+        MPPTools.uploadFileToBintray("${bintrayUrl}", "${bintrayRepo}",
+                buildNumber, "${bintrayPackage}", "${target}/${buildNumber}/${fileName}",
+                "${buildDir.absolutePath}/${fileName}", user, password)
+    }
+}
+
 task bench(type:Exec) {
     dependsOn jvmRun
     dependsOn konanRun
     def extension = MPPTools.getNativeProgramExtension()
     def analyzer = MPPTools.findFile("${analyzerTool}${extension}", "${rootBuildDirectory}/${analyzerToolDirectory}")
     if (analyzer != null) {
-        commandLine "${analyzer}", "${buildDir.absolutePath}/${nativeJson}", "${buildDir.absolutePath}/${jvmJson}"
+        commandLine "${analyzer}", "-r", "text", "-r", "teamcity", "${buildDir.absolutePath}/${nativeJson}", "${buildDir.absolutePath}/${jvmJson}"
     } else {
         println("No analyzer $analyzerTool found in subdirectories of ${rootBuildDirectory}/${analyzerToolDirectory}")
     }
+}
+
+// Produce and send slack report.
+task slackReport(type: RegressionsReporter) {
+    // Create folder for report (root Kotlin project settings make create report in separate folder).
+    def reportDirectory = outputReport.substring(0, outputReport.lastIndexOf("/"))
+    mkdir reportDirectory
+    currentBenchmarksReportFile = "${buildDir.absolutePath}/${nativeJson}"
+    analyzer = MPPTools.findFile("${analyzerTool}${MPPTools.getNativeProgramExtension()}",
+            "${rootBuildDirectory}/${analyzerToolDirectory}")
+    htmlReport = outputReport
+    defaultBranch = project.hasProperty('kotlin.native.default.branch') ? property('kotlin.native.default.branch') : "master"
```

Comment:
It's possible to use the `findProperty` method to get rid duplicating the property access:
```
defaultBranch = project.findProperty('kotlin.native.default.branch') ?: "master"
```